### PR TITLE
feat(ui): add a more prominent manage button in project dropdown

### PR DESCRIPTION
**Pull request by ckj**
screen recording 2019-02-05 at 04 30 pm

### CODE REVIEW

File: src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx [hunk 1]
```diff
@@ -389,7 +390,19 @@ class DropdownAutoCompleteMenu extends React.Component {
                       <InputLoadingWrapper>
                         {busy && <LoadingIndicator size={16} mini />}
                       </InputLoadingWrapper>
-                    </Flex>
+                      {menuProps &&
+                        menuProps.showSettingsLink && (
+                          <div>
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx]: why div?

File: src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx [hunk 2]
```diff
@@ -499,22 +512,27 @@ const AutoCompleteRoot = styled(({isOpen, ...props}) => <div {...props} />)`
...
+  border: 1px solid transparent;
 
   &,
   &:focus,
   &:active,
   &:hover {
-    border: 1px solid transparent;
-    border-bottom: 1px solid ${p => p.theme.borderLight};
-    border-radius: ${p => `${p.theme.borderRadius} ${p.theme.borderRadius} 0 0`};
+    border: 0;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx]: I don't totally understand what is going on here. we don't need any of this stuff anymore?

**Review by Chrissy — APPROVED** [review]
There are a few odd styles in here I don't totally understand but I trust u

File: src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx]: If I remove `div` the button becomes the flex-child, and thus full height of the container. We could use `Box` instead, but that doesn't seem worth the import.

File: src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx (hunk 2 above)
Comment (ckj) [reply src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx]: I moved them to the parent flexbox (StyledInputWrapper) so the border spans the full width of the dropdown and not just the Input.

**Pull request merged**